ppr-ui/src/components/dialogs/StaffPaymentDialog.vue

```diff
@@ -7,7 +7,7 @@
     <template v-slot:content>
       <staff-payment-component
         :staffPaymentData="staffPaymentData"
-        :validate="validate"
+        :validate="true"
```

Comment:
I think you meant to put this back to 'validate' right? Otherwise it doesn't look like 'validate' is being used for anything so you can probably remove it from the code below?

Reply:
Removed it